### ISSUE TITLE
AMS interface font color contrast is insufficient

### ISSUE THREAD

**Issue by SiliciumMan** [bambulab/BambuStudio#7792]
### Bambu Studio Version

2.1.1.52

### Where is the application from?

Bambu Lab Official website

### OS version

Windows 11

### Printer

P1S

### How to reproduce

1. Choose a black filament
2. Use it until it's half gone

### Actual results

You won't be able to distinguish the names of filaments

<img width="396" height="259" alt="Image" src="https://github.com/user-attachments/assets/55fb0c58-0007-4b05-881d-aaccc6cef429" />

### Expected results

Use a font color different from the background and filament color, or display preset information about the material when the mouse hovers over it.
Or change the colour of the background.

### Project file & Debug log uploads

NA

### Checklist of files to include

- [ ] Log file
- [ ] Project file

**Comment by qusheng-bambu** [comment]
Thank you very much for your feedback! This should be a feature. We will consider your suggestions and provide users with a more user-friendly interface
